zb-db/src/main/java/io/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -64,20 +66,25 @@ protected ZeebeTransactionDb(
     closables.add(prefixReadOptions);
     defaultReadOptions = new ReadOptions();
     closables.add(defaultReadOptions);
-    defaultWriteOptions = new WriteOptions();
+    defaultWriteOptions = new WriteOptions().setDisableWAL(rocksDbConfiguration.isWalDisabled());
+    LOG.info("RocksDB WAL is disabled: " + defaultWriteOptions.disableWAL());
```

Comment:
Is this really what we want to log as info? What is the difference between the log we have in the begin were we print all configs. Plus if you want to keep it I would like to have something like:

```
LOG.info("RocksDB WAL is {} ", rocksDbConfiguration.isWalDisabled() ? "disabled" : "enabled");
```

Reply:
No, sorry, that was a debugging left over :see_no_evil: